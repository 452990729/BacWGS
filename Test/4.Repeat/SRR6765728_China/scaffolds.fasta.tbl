==================================================
file name: scaffolds.fasta          
sequences:          2258
total length:    1957732 bp  (1957732 bp excl N/X-runs)
GC level:         52.57 %
bases masked:      12492 bp ( 0.64 %)
==================================================
               number of      length   percentage
               elements*    occupied  of sequence
--------------------------------------------------
SINEs:                0            0 bp    0.00 %
      ALUs            0            0 bp    0.00 %
      MIRs            0            0 bp    0.00 %

LINEs:                0            0 bp    0.00 %
      LINE1           0            0 bp    0.00 %
      LINE2           0            0 bp    0.00 %
      L3/CR1          0            0 bp    0.00 %

LTR elements:         0            0 bp    0.00 %
      ERVL            0            0 bp    0.00 %
      ERVL-MaLRs      0            0 bp    0.00 %
      ERV_classI      0            0 bp    0.00 %
      ERV_classII     0            0 bp    0.00 %

DNA elements:         0            0 bp    0.00 %
     hAT-Charlie      0            0 bp    0.00 %
     TcMar-Tigger     0            0 bp    0.00 %

Unclassified:       177        13902 bp    0.71 %

Total interspersed repeats:    13902 bp    0.71 %


Small RNA:            0            0 bp    0.00 %

Satellites:           0            0 bp    0.00 %
Simple repeats:       0            0 bp    0.00 %
Low complexity:       0            0 bp    0.00 %
==================================================

* most repeats fragmented by insertions or deletions
  have been counted as one element
                                                      

The query species was assumed to be homo          
RepeatMasker Combined Database: Dfam_3.0
                                         
run with blastp version 3.0PE-AB [2009-10-30] [linux26-x64-I32LPF64 2009-11-17T18:52:53]
The query was compared to unclassified sequences in ".../RepBase.fasta"

