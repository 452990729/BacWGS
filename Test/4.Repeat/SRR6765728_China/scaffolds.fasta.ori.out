   234 10.26 0.00 2.56 NODE_1033_length_699_cov_2.453416       135       174    (525) C          DNA-6-10_NV     Unspecified  (1239)    1165    1127    
   226 18.75 1.56 8.33 NODE_103_length_2322_cov_2.448610      1491      1554    (768) C        Gypsy-2_CSa-I     Unspecified  (2721)    3000    2941    
   322 23.61 0.00 0.00 NODE_103_length_2322_cov_2.448610      1842      1913    (409) C         tRNA-Tyr-TAC     Unspecified     (0)      76       5    
   259 21.19 0.00 5.63 NODE_103_length_2322_cov_2.448610      1847      1921    (401) +       CryptonS-23_PI     Unspecified    2589    2659   (242)    
   249 17.23 5.08 1.64 NODE_104_length_2314_cov_3.061974       267       325   (1989) +        Mariner-4_LMi     Unspecified    1137    1197   (429)    
   278 17.65 0.00 0.00 NODE_104_length_2314_cov_3.061974       278       328   (1986) +        Mariner-1_PSo     Unspecified    1201    1251   (410)    
   657 20.96 0.60 0.60 NODE_1052_length_687_cov_3.292722        12       179    (508) +        EnSpm-29_CCri     Unspecified    4541    4708  (6265)    
   230 32.75 0.00 5.44 NODE_106_length_2295_cov_1.877232      1822      1976    (319) +        Sola1-2C_CCri     Unspecified    2813    2959  (9425)    
   235 15.23 9.38 7.69 NODE_1089_length_664_cov_2.546798       538       601     (63) +       tRNA-SerCGA_OS     Unspecified      11      75     (7)    
   227 6.06 3.03 0.00 NODE_10_length_4419_cov_2.320577      3351      3383   (1036) +           DNA2-25_CB     Unspecified     379     412   (574)    
   235 26.58 1.52 3.08 NODE_1107_length_657_cov_2.559801       369       434    (223) C         BEL-615_AA-I     Unspecified  (4329)    1854    1790    
   312 17.54 0.00 0.00 NODE_1123_length_649_cov_2.373737       365       421    (228) C         tRNA-Lys-AAA     Unspecified    (12)      64       8    
   242 27.65 0.00 7.50 NODE_1129_length_647_cov_2.331081        42       127    (520) C        tRNA-Ser-TCA_     Unspecified     (0)      85       6    
   235 26.50 6.93 2.86 NODE_1169_length_625_cov_1.978947       282       382    (243) +            hAT-10_XT     Unspecified      59     163 (10981)    
   225 20.00 0.00 0.00 NODE_1179_length_619_cov_1.998227        36        75    (544) C        Copia-56_BD-I     Unspecified   (457)    4757    4718    
   225 27.40 2.74 0.00 NODE_1200_length_604_cov_2.167577       486       558     (46) C       Gypsy-108_AA-I     Unspecified  (2703)    1916    1842    
   240 18.46 10.77 0.00 NODE_1223_length_588_cov_2.392120       464       528     (60) +           DNA-7-5_NV     Unspecified     677     748  (2951)    
   225 19.70 2.94 2.94 NODE_1238_length_583_cov_1.916667        51       118    (465) C         LINE1-N1I_OS     Unspecified   (978)     526     459    
   287 25.37 2.99 0.00 NODE_1277_length_568_cov_1.980507       146       212    (356) +        tRNAIleATT_CB     Unspecified       3      71    (10)    
   227 22.92 0.00 0.00 NODE_1312_length_555_cov_1.766000       350       397    (158) +         Jockey-N4_CQ     Unspecified    1233    1280   (513)    
   234 23.72 3.33 1.64 NODE_1328_length_545_cov_1.512245       272       331    (214) +          ERV1-2-I_BT     Unspecified    5068    5128  (1158)    
   248 20.00 1.64 1.64 NODE_1346_length_531_cov_2.478992       434       494     (37) C     Harbinger-2C_CPB     Unspecified  (4068)     444     384    
   227 19.25 14.29 1.05 NODE_134_length_2078_cov_2.001483       732       815   (1263) C           SART-5_APi     Unspecified  (2409)    1996    1902    
   242 10.87 6.52 0.00 NODE_134_length_2078_cov_2.001483      1191      1236    (842) C            L1-30_AAe     Unspecified  (2173)    3364    3316    
   226 27.42 3.23 0.00 NODE_1360_length_525_cov_1.902128        44       105    (420) C        Zator-1N1_LCh     Unspecified   (651)     277     214    
   230 12.51 0.00 2.50 NODE_137_length_2063_cov_2.192231       605       645   (1418) +        Polinton-2_TC     Unspecified   16912   16951    (30)    
   226 17.65 13.73 0.00 NODE_13_length_4332_cov_2.525836      3499      3549    (783) +          BEL-15_AA-I     Unspecified    1888    1945  (3423)    
   255 23.66 0.00 4.17 NODE_1404_length_506_cov_2.230599       160       234    (272) C              SINE_SM     Unspecified    (76)      85      14    
   312 17.24 0.00 0.00 NODE_1404_length_506_cov_2.230599       173       230    (276) C         tRNA-Lys-AAG     Unspecified    (11)      65       8    
   240 21.01 10.26 2.38 NODE_140_length_2055_cov_2.482500      1477      1554    (501) +           Tad1-34_BG     Unspecified    5186    5269   (741)    
   230 21.84 6.45 6.45 NODE_1440_length_494_cov_2.369021       236       328    (166) C        Copia-73_ZM-I     Unspecified  (2352)    3778    3686    
   225 18.75 2.08 0.00 NODE_1493_length_464_cov_2.017115       370       417     (47) C        Copia-84_SB-I     Unspecified  (1712)    2805    2757    
   228 23.72 3.70 6.33 NODE_1550_length_438_cov_1.832898       169       249    (189) +       Gypsy-9N1-I_XT     Unspecified     384     462  (2680)    
   245 19.61 0.00 0.00 NODE_1572_length_430_cov_1.584000       283       333     (97) +    Gypsy-10_DVir-LTR     Unspecified    1962    2012    (26)    
   245 13.21 7.55 0.00 NODE_1619_length_416_cov_1.418283       142       194    (222) +          Jockey-4_DK     Unspecified    2694    2750  (1371)    
   228 8.13 0.00 5.41 NODE_1664_length_399_cov_2.796512       330       368     (31) +        Copia-91_VV-I     Unspecified    3680    3716   (904)    
   234 11.25 0.00 11.11 NODE_167_length_1901_cov_2.264355       898       947    (954) C       Gypsy-28_PSo-I     Unspecified  (3870)    2583    2539    
   234 14.29 0.00 0.00 NODE_169_length_1894_cov_2.008700      1451      1485    (409) C        Gypsy-13_OD-I     Unspecified  (3810)    3383    3349    
   226 10.81 2.70 0.00 NODE_178_length_1847_cov_2.068080       417       453   (1394) +       Gypsy-258_AA-I     Unspecified    2996    3033  (2219)    
   325 20.97 0.00 0.00 NODE_1830_length_341_cov_1.416084       132       193    (148) +         tRNA-Tyr-TAC     Unspecified       6      67     (9)    
   228 25.95 8.42 10.75 NODE_1834_length_338_cov_2.724382       142       236    (102) C     Gypsy-71H_CCri-I     Unspecified  (7980)    3148    3056    
   238 13.47 1.85 3.77 NODE_184_length_1806_cov_2.535694         1        54   (1752) +    piggyBacX-20_CCri     Unspecified    1484    1536  (3736)    
   229 18.18 0.00 0.00 NODE_1859_length_333_cov_3.111511       185       228    (105) +             GYZMA1_I     Unspecified    3072    3115  (1695)    
   232 14.52 12.50 1.61 NODE_185_length_1805_cov_2.230286       630       685   (1120) C           LTR23_Tbel     Unspecified   (221)     271     210    
   233 24.07 7.41 0.00 NODE_1944_length_304_cov_1.903614       219       272     (32) C          EnSpm-11_ZM     Unspecified   (110)    7816    7759    
   241 12.49 4.08 2.00 NODE_1971_length_297_cov_1.983471       227       275     (22) +        Gypsy-15_CQ-I     Unspecified    2275    2324  (2932)    
  2642 0.34 0.00 0.00 NODE_1981_length_293_cov_2.109244         1       293      (0) C            L1HS_5end     Unspecified   (908)    1228     936    
   225 15.91 4.55 0.00 NODE_1994_length_289_cov_2.525641       143       186    (103) +          Gypsy6-I_VC     Unspecified    3533    3578  (1352)    
   232 11.75 7.55 3.64 NODE_199_length_1748_cov_2.886592      1024      1076    (672) +           P-1_Dpulex     Unspecified    4330    4384   (323)    
   228 26.45 0.00 3.77 NODE_1_length_8085_cov_2.795143      4788      4842   (3243) +          BEL-88_CQ-I     Unspecified    5238    5290   (483)    
   226 20.00 10.77 0.00 NODE_1_length_8085_cov_2.795143      6574      6638   (1447) C         BEL-599_AA-I     Unspecified  (4832)     606     535    
   236 17.43 8.96 5.80 NODE_2018_length_280_cov_2.422222        85       151    (129) +       Gypsy-12_Sit-I     Unspecified    2605    2673  (1729)    
   240 14.29 0.00 0.00 NODE_201_length_1745_cov_2.020710      1649      1683     (62) C       Gypsy-35_AIp-I     Unspecified  (4389)     115      81    
   242 19.69 2.20 10.71 NODE_2023_length_279_cov_2.245536        95       185     (94) +      Mariner-N1B_LMi     Unspecified     186     269   (657)    
   307 26.59 0.00 1.27 NODE_2023_length_279_cov_2.245536        99       178    (101) +           tRNAThr_CB     Unspecified       6      84     (0)    
   235 16.14 3.08 4.69 NODE_2023_length_279_cov_2.245536       206       270      (9) +           SINE2-2_SP     Unspecified      10      73   (178)    
   238 2.80 0.00 18.92 NODE_2033_length_276_cov_2.289593        18        61    (215) +          LINE1-31_ZM     Unspecified    1047    1083  (2599)    
   230 26.39 4.17 0.00 NODE_204_length_1734_cov_2.550923       752       823    (911) +        Jockey-2_DGri     Unspecified     664     738  (5294)    
   241 20.97 0.00 0.00 NODE_204_length_1734_cov_2.550923       762       823    (911) +            Ag-Jock-1     Unspecified    1403    1464  (3261)    
   250 19.02 6.82 4.44 NODE_206_length_1727_cov_1.804426       527       614   (1113) +           SINE2-2_SP     Unspecified       6      95   (156)    
   266 19.70 0.00 3.28 NODE_206_length_1727_cov_1.804426       531       593   (1134) +           SINE2-1_VC     Unspecified       6      66   (602)    
   242 15.38 0.00 0.00 NODE_210_length_1701_cov_1.963548      1135      1173    (528) +      Polinton-1N1_DR     Unspecified    7565    7603  (7866)    
  1551 0.56 0.00 0.00 NODE_2185_length_225_cov_3.247059        47       225      (0) C            L1HS_5end     Unspecified   (737)    1399    1221    
   246 4.00 0.00 0.00 NODE_2256_length_62_cov_1.000000         5        54      (8) +        Polinton-1_CB     Unspecified    6868    6917  (9716)    
   246 4.00 0.00 0.00 NODE_2256_length_62_cov_1.000000        12        61      (1) +        Polinton-1_CB     Unspecified    6868    6917  (9716)    
   242 21.93 0.00 7.27 NODE_22_length_3984_cov_3.080173       267       325   (3659) C        Gypsy-1_CDC-I     Unspecified  (1152)    4213    4159    
   322 20.39 6.00 1.92 NODE_22_length_3984_cov_3.080173      1503      1602   (2382) +              CACTA-N     Unspecified     585     688   (299)    
   230 21.15 1.92 0.00 NODE_235_length_1631_cov_1.773477      1139      1190    (441) C            Tad1-1_BG     Unspecified   (286)    6050    5998    
   244 21.74 1.56 6.56 NODE_236_length_1628_cov_2.108709      1014      1077    (551) C    Harbinger-17_CCri     Unspecified   (495)    2269    2209    
   603 32.39 1.96 1.96 NODE_236_length_1628_cov_2.108709      1374      1628      (0) +        EnSpm-29_CCri     Unspecified    4269    4523  (6450)    
   236 12.20 12.20 0.00 NODE_253_length_1588_cov_2.770385      1542      1582      (6) +        Copia-75_AA-I     Unspecified    1171    1216  (3142)    
   227 27.78 1.37 1.37 NODE_26_length_3834_cov_2.571315      2730      2802   (1032) C          Copia3-I_CR     Unspecified  (1450)    3890    3818    
   235 3.57 0.00 0.00 NODE_279_length_1527_cov_1.894022        33        60   (1467) +       Copia-129_SB-I     Unspecified    1355    1382  (4271)    
   225 27.93 4.35 1.41 NODE_285_length_1517_cov_1.978112       366       434   (1083) C            LTR-1_SBi     Unspecified   (285)    4453    4383    
   238 15.92 0.00 2.27 NODE_291_length_1504_cov_2.245687        53        97   (1407) C           BEL30-I_DR     Unspecified  (4646)    2002    1959    
   237 22.22 0.00 0.00 NODE_296_length_1493_cov_1.995132      1365      1409     (84) +         ERV1-4_TSy-I     Unspecified    4263    4307   (807)    
   243 25.93 0.00 0.00 NODE_29_length_3746_cov_1.964779       833       886   (2860) +       Copia-27_PIT-I     Unspecified    2733    2786  (2231)    
   256 23.73 1.69 0.00 NODE_304_length_1482_cov_2.184303       472       530    (952) C        Gypsy-3_CFl-I     Unspecified  (3706)    1746    1687    
   230 29.89 4.60 0.00 NODE_304_length_1482_cov_2.184303      1010      1096    (386) C        Gypsy-73_MN-I     Unspecified  (1223)    5516    5426    
   239 24.57 0.00 1.75 NODE_309_length_1472_cov_2.315455       196       253   (1219) C       Gypsy-15_SCH-I     Unspecified  (6626)    4663    4607    
   231 24.19 0.00 0.00 NODE_30_length_3717_cov_2.176133       768       829   (2888) C     BivaV-SINE3_CoAn     Unspecified   (206)      64       3    
   240 32.88 0.00 1.37 NODE_30_length_3717_cov_2.176133       860       933   (2784) C         tRNA-Arg2_OS     Unspecified     (0)      73       1    
   233 6.67 0.00 0.00 NODE_30_length_3717_cov_2.176133       967       996   (2721) C        FEILAI-1B_AAe     Unspecified   (210)      79      50    
   230 14.65 0.00 2.44 NODE_31_length_3697_cov_2.760297      2893      2934    (763) C        Copia-1_ALY-I     Unspecified  (4573)     225     185    
   251 22.24 1.33 4.11 NODE_31_length_3697_cov_2.760297      3224      3298    (399) +       Mariner3-4_LMi     Unspecified     172     244  (2787)    
   256 23.84 0.00 3.17 NODE_31_length_3697_cov_2.760297      3234      3298    (399) C             TE-3_LVa     Unspecified  (1778)     882     820    
   239 17.39 2.17 0.00 NODE_320_length_1439_cov_2.726156       942       987    (452) +       Helitron-1_AEc     Unspecified    5637    5683  (2124)    
   284 26.23 6.86 2.83 NODE_338_length_1390_cov_2.560300      1229      1330     (60) C            GarnAlu3B     Unspecified   (147)     113       8    
   248 24.08 1.82 1.82 NODE_353_length_1367_cov_2.041921       503       557    (810) C       Gypsy-160_AA-I     Unspecified  (2493)    3594    3540    
   227 13.05 2.13 2.13 NODE_354_length_1366_cov_2.196034      1000      1046    (320) +        Penelope-1_DR     Unspecified     524     570  (3876)    
   226 15.00 7.50 0.00 NODE_369_length_1343_cov_1.837733      1238      1277     (66) C           DNA5-1_PSt     Unspecified    (44)     377     335    
   250 30.23 3.64 7.56 NODE_372_length_1332_cov_2.119029       472       718    (614) +          CRE-15_CCri     Unspecified    2090    2327   (121)    
   231 15.64 4.41 5.97 NODE_38_length_3490_cov_2.610189      1289      1356   (2134) C             Tx1-4_DR     Unspecified  (2213)    3322    3256    
   236 26.67 5.33 0.00 NODE_38_length_3490_cov_2.610189      1801      1875   (1615) +         Ogre-PT2_LTR     Unspecified    2017    2095   (680)    
   229 7.92 0.00 5.26 NODE_38_length_3490_cov_2.610189      2060      2099   (1391) C        Copia-6_ALY-I     Unspecified  (1436)    2620    2583    
   249 15.87 14.29 0.00 NODE_38_length_3490_cov_2.610189      2147      2209   (1281) +            RTE-7_LCh     Unspecified    2532    2603  (1403)    
   230 17.01 6.25 2.00 NODE_392_length_1309_cov_1.971292       449       496    (813) +        Copia-7_PGr-I     Unspecified    1154    1203  (3759)    
   227 31.11 2.17 6.82 NODE_392_length_1309_cov_1.971292       511       648    (661) C        Jockey-3_DGri     Unspecified  (1212)    3501    3370    
   261 30.29 0.00 2.75 NODE_392_length_1309_cov_1.971292       538       649    (660) +               R1_DYa     Unspecified     721     829  (4684)    
   253 11.43 0.00 0.00 NODE_395_length_1306_cov_2.329337       546       580    (726) C       Gypsy-69_PAb-I     Unspecified  (4037)     913     879    
   229 20.83 8.33 0.00 NODE_396_length_1306_cov_2.146283       378       425    (881) C          Nimb-31_LMi     Unspecified  (1066)    4875    4824    
   230 24.00 0.00 0.00 NODE_403_length_1297_cov_2.557971      1166      1215     (82) +           hATm-41_HM     Unspecified     185     234  (2750)    
   231 18.02 1.92 3.92 NODE_412_length_1290_cov_2.336842       253       304    (986) C           hAT-42_Lch     Unspecified  (1459)    2016    1966    
   230 6.65 10.42 6.00 NODE_430_length_1259_cov_2.241694        68       115   (1144) C                R7Ag2     Unspecified  (3780)    2810    2761    
   225 19.55 8.08 7.00 NODE_477_length_1180_cov_2.351111       864       962    (218) C              P-30_HM     Unspecified  (3958)    2315    2216    
   239 20.52 13.51 1.20 NODE_48_length_3180_cov_3.223040      1448      1521   (1659) +        Helitron-1_SP     Unspecified    8027    8109  (1991)    
   225 9.28 13.64 2.04 NODE_490_length_1165_cov_2.574775       182       225    (940) C        Gypsy-1_PGr-I     Unspecified  (2056)    5764    5716    
   240 22.45 2.04 0.00 NODE_490_length_1165_cov_2.574775       490       538    (627) C     Gypsy-11_CFl-LTR     Unspecified   (909)     306     257    
   242 22.42 0.00 1.72 NODE_492_length_1165_cov_1.801802       606       664    (501) +       Copia-39_AIp-I     Unspecified     222     279  (4631)    
   230 17.87 1.72 3.51 NODE_499_length_1154_cov_2.083712        47       104   (1050) +       Gypsy-15_ATr-I     Unspecified   11118   11174  (2772)    
   316 21.59 15.38 3.81 NODE_4_length_7157_cov_3.324838      4252      4511   (2646) +             hAT-4_RC     Unspecified     938    1226  (3888)    
   436 28.76 1.47 4.55 NODE_4_length_7157_cov_3.324838      4265      4468   (2689) C       Gypsy-32_ADe-I     Unspecified  (9476)    2053    1856    
   385 29.68 0.00 3.54 NODE_4_length_7157_cov_3.324838      4282      4515   (2642) C        MarinerL-1_CR     Unspecified (14376)    1612    1387    
   453 23.16 0.00 10.84 NODE_4_length_7157_cov_3.324838      4328      4511   (2646) C       Gypsy-32_ADe-I     Unspecified  (9464)    2065    1900    
   392 23.17 0.00 0.00 NODE_501_length_1153_cov_2.651184       460       541    (612) C              ALPINE2     Unspecified    (56)      88       7    
   245 20.70 0.00 1.72 NODE_501_length_1153_cov_2.651184       562       620    (533) C         tRNA-Lys-AAA     Unspecified    (11)      65       8    
   239 17.57 0.00 3.51 NODE_501_length_1153_cov_2.651184       569       627    (526) C           SINE2-7_SP     Unspecified   (149)      62       6    
   231 17.39 2.17 0.00 NODE_508_length_1148_cov_2.735590       674       719    (429) +        Copia-92_SB-I     Unspecified    4037    4083   (308)    
   230 16.68 2.00 4.08 NODE_523_length_1130_cov_2.275349       416       465    (665) +         Gypsy65-ZM_I     Unspecified    5053    5101   (512)    
   229 13.15 0.00 17.02 NODE_525_length_1129_cov_2.257914       128       182    (947) +          Gypsy5-VV_I     Unspecified    4384    4430   (132)    
   269 12.50 5.36 0.00 NODE_53_length_3084_cov_2.179597        28        83   (3001) +        Gypsy-91_ZM-I     Unspecified      15      73  (4549)    
   234 14.63 5.71 12.12 NODE_544_length_1096_cov_2.370797       111       180    (916) C       Harbinger-2_XT     Unspecified  (2137)   10669   10604    
   233 14.29 0.00 2.38 NODE_593_length_1039_cov_2.147358       369       411    (628) +          ATCopia34_I     Unspecified     658     699  (3561)    
   235 17.54 8.06 8.06 NODE_599_length_1036_cov_2.194699       303       364    (672) +     Helitron-1_PTrit     Unspecified    4112    4173   (257)    
   225 32.00 0.00 0.00 NODE_5_length_6840_cov_3.507148      2029      2128   (4712) +         SSU-rRNA_Lma     Unspecified    1163    1262   (875)    
   661 32.22 7.87 3.25 NODE_5_length_6840_cov_3.507148      2068      2715   (4125) +         SSU-rRNA_Tps     Unspecified     981    1657   (147)    
   357 28.43 5.51 11.36 NODE_5_length_6840_cov_3.507148      2457      2837   (4003) +     SSU-rRNA_Giardia     Unspecified    1092    1452    (12)    
   266 27.37 3.53 1.15 NODE_5_length_6840_cov_3.507148      2943      3027   (3813) +           SINE2-2_SP     Unspecified       2      88   (163)    
   296 23.73 0.00 0.00 NODE_5_length_6840_cov_3.507148      2953      3011   (3829) +          tRNA-Asn_OS     Unspecified       8      66    (14)    
   345 24.18 0.00 1.10 NODE_5_length_6840_cov_3.507148      3028      3119   (3721) +              ALPINE1     Unspecified       4      94    (33)    
   252 27.08 4.95 4.95 NODE_5_length_6840_cov_3.507148      3035      3135   (3705) +           SINE2-1_PL     Unspecified      13     113   (135)    
   304 25.28 0.00 1.10 NODE_5_length_6840_cov_3.507148      3619      3710   (3130) +         LSU-rRNA_Hca     Unspecified      35     125  (3530)    
   321 24.14 0.00 0.00 NODE_5_length_6840_cov_3.507148      3631      3717   (3123) +         LSU-rRNA_Ddi     Unspecified      23     109  (3132)    
   315 19.99 4.55 3.37 NODE_5_length_6840_cov_3.507148      3848      3935   (2905) +         LSU-rRNA_Ddi     Unspecified     293     381  (2860)    
   390 23.96 0.00 0.00 NODE_5_length_6840_cov_3.507148      3870      3965   (2875) +         LSU-rRNA_Hca     Unspecified     338     433  (3222)    
   292 24.50 0.99 3.03 NODE_5_length_6840_cov_3.507148      3893      3993   (2847) +         LSU-rRNA_Ldo     Unspecified      11     109   (881)    
   264 29.79 3.23 2.56 NODE_5_length_6840_cov_3.507148      4116      4270   (2570) +     LSU-rRNA_Giardia     Unspecified     518     673  (2017)    
  1043 22.59 3.09 4.38 NODE_5_length_6840_cov_3.507148      4402      4725   (2115) C        Copia-18_BD-I     Unspecified    (64)    4768    4449    
   279 27.42 4.92 14.80 NODE_5_length_6840_cov_3.507148      5047      5290   (1550) +     LSU-rRNA_Giardia     Unspecified    1472    1694   (996)    
   713 25.39 2.73 5.12 NODE_5_length_6840_cov_3.507148      5214      5433   (1407) +     LSU-rRNA_Giardia     Unspecified    1616    1830   (860)    
   475 28.94 6.25 8.72 NODE_5_length_6840_cov_3.507148      5217      5568   (1272) +         LSU-rRNA_Hca     Unspecified    2411    2754   (901)    
   567 27.66 6.42 6.42 NODE_5_length_6840_cov_3.507148      5645      6127    (713) +         LSU-rRNA_Hsa     Unspecified    4123    4605   (430)    
   598 30.00 4.52 4.52 NODE_5_length_6840_cov_3.507148      5649      6046    (794) +         LSU-rRNA_Dme     Unspecified    3042    3439   (461)    
   492 31.57 4.32 4.70 NODE_5_length_6840_cov_3.507148      5656      6210    (630) +         LSU-rRNA_Cel     Unspecified    2699    3251   (258)    
   228 22.24 1.79 3.64 NODE_603_length_1033_cov_1.829243       746       801    (232) C          BEL-20_AG-I     Unspecified  (1882)    3429    3375    
   230 10.54 0.00 2.63 NODE_613_length_1023_cov_1.960744       852       890    (133) +              P-22_HM     Unspecified    3093    3130    (35)    
   225 22.03 0.00 4.00 NODE_63_length_2833_cov_2.200504      1618      1669   (1164) +            Loa-1_SIn     Unspecified    4261    4310   (816)    
   476 32.39 1.39 2.11 NODE_694_length_939_cov_2.038462       653       939      (0) C          MuDR-3_CCri     Unspecified  (6710)    2514    2230    
   230 14.30 0.00 2.86 NODE_716_length_916_cov_1.825784       499       534    (382) +          BEL-74_CQ-I     Unspecified     312     346  (5823)    
   227 15.07 9.26 1.72 NODE_736_length_899_cov_2.248815        40        93    (806) +         BEL-626_AA-I     Unspecified    6179    6236   (673)    
   235 19.61 12.28 1.59 NODE_741_length_894_cov_2.476758       787       843     (51) C              R1-1_DT     Unspecified   (164)    4000    3938    
   225 17.65 9.80 0.00 NODE_74_length_2645_cov_2.382625       815       865   (1780) +               LINE-1     Unspecified    3427    3482  (1338)    
   228 12.50 25.00 0.00 NODE_750_length_887_cov_2.235577       768       823     (64) C         BEL-22_DTa-I     Unspecified  (3264)    4849    4780    
   236 29.07 15.12 0.00 NODE_75_length_2642_cov_2.406262       371       456   (2186) C        Gypsy-30_OD-I     Unspecified  (5464)    1113    1015    
   227 18.75 2.08 0.00 NODE_782_length_864_cov_1.948084       634       681    (183) +       Chapaev-1N1_NV     Unspecified     535     583  (2001)    
   240 19.53 0.00 2.44 NODE_78_length_2622_cov_2.156213       160       201   (2421) +           SART-2_PXu     Unspecified    3020    3060  (2380)    
   267 24.33 1.32 2.67 NODE_795_length_857_cov_1.842893       761       836     (21) +        Gypsy-6_Ano-I     Unspecified     378     452  (4892)    
   235 20.03 1.47 4.55 NODE_7_length_5281_cov_3.893035       117       184   (5097) +        Gypsy-18_AA-I     Unspecified    2860    2925  (1460)    
   227 18.18 6.82 0.00 NODE_7_length_5281_cov_3.893035      3464      3507   (1774) +      Mariner-5_PTrit     Unspecified     622     668  (2915)    
   233 21.95 0.00 0.00 NODE_7_length_5281_cov_3.893035      4309      4349    (932) +        Gypsy-1_CMe-I     Unspecified   10215   10255  (2155)    
   234 18.52 14.81 0.00 NODE_828_length_834_cov_2.192555       170       223    (611) C         BEL-617_AA-I     Unspecified  (2199)    3649    3588    
   235 26.39 2.70 2.70 NODE_82_length_2557_cov_1.962430      1884      1957    (600) +       Gypsy-14_DAn-I     Unspecified       6      79  (6026)    
   312 27.40 0.00 0.00 NODE_82_length_2557_cov_1.962430      1892      1964    (593) C         tRNA-Arg2_OS     Unspecified     (0)      73       1    
   246 16.28 2.33 0.00 NODE_835_length_831_cov_1.603093       109       151    (680) +         EnSpm-8_CCri     Unspecified   10881   10924   (765)    
   239 25.00 0.00 0.00 NODE_852_length_820_cov_2.950327       242       297    (523) C            ManaSINE1     Unspecified   (261)      70      15    
   256 23.73 0.00 1.69 NODE_852_length_820_cov_2.950327       334       393    (427) C          tRNA-Asn_OS     Unspecified    (14)      66       8    
   227 7.46 4.35 2.86 NODE_85_length_2529_cov_2.308812        69       137   (2392) C        TguERVL1a_LTR     Unspecified   (421)     256     187    
   228 11.12 0.00 1.85 NODE_85_length_2529_cov_2.308812        83       137   (2392) C     Harbinger-8_CCri     Unspecified  (2830)    2574    2521    
   225 11.51 0.00 11.36 NODE_86_length_2513_cov_2.644020      2344      2392    (121) +            DNA8-1_CQ     Unspecified     154     197  (1960)    
   235 21.61 1.19 11.84 NODE_890_length_802_cov_2.016064       648       731     (71) +            hAT-51_HM     Unspecified    3549    3624   (210)    
   257 19.76 3.57 3.57 NODE_890_length_802_cov_2.016064       670       753     (49) +             Ves2B_ML     Unspecified       7      90   (162)    
   317 18.97 0.00 0.00 NODE_890_length_802_cov_2.016064       675       732     (70) +         SINE2-2B_SSa     Unspecified       9      66   (116)    
   226 29.49 1.28 0.00 NODE_89_length_2482_cov_2.310260      2321      2398     (84) C          BEL-2_LMi-I     Unspecified   (927)    4938    4860    
   273 20.75 5.68 6.90 NODE_89_length_2482_cov_2.310260      2331      2418     (64) +          BEL-2_LMi-I     Unspecified    5223    5309   (556)    
   233 14.27 8.33 6.56 NODE_968_length_739_cov_2.596491       184       243    (496) C       Helitron-3_ALy     Unspecified  (8835)    6161    6101    
   243 17.78 0.00 0.00 NODE_96_length_2400_cov_2.190192      2157      2201    (199) +       Helitron-1_Crp     Unspecified    4193    4237   (959)    
   234 25.81 0.00 4.55 NODE_973_length_736_cov_2.249633       239       307    (429) +        MarinerL-1_CR     Unspecified    3248    3313 (12675)    
